### ISSUE TITLE
Icons not showing when using IE11

### ISSUE THREAD

**Issue by linuxpete** [juliushaertl/apporder#51]
### Steps to reproduce
1. Enable AppOrder
2. Open IE 11 and Login
3. See that appicons are not showing, but the links are available on the places of the icons!
4. Disable Apporder
5. Login again
6. See that appicons are showing

Firefox and Chrome are working like advertised

### Expected behaviour
See : AppOrder_WithIcons.png
Icons must show in IE. 

### Actual behaviour
See : AppOrder_NoIcons.png
App Icons don't show when using IE.

![apporder_noicons](https://user-images.githubusercontent.com/7695965/30745365-1b056762-9fa6-11e7-8bd0-662d0941dad4.png)
![apporder_withicons](https://user-images.githubusercontent.com/7695965/30745366-1b1064a0-9fa6-11e7-89ed-88bb41d30479.png)

### General server configuration

**Operating system:** Linux mo 4.1.19-v7+ #858 SMP Tue Mar 15 15:56:00 GMT 2016 armv7l

**Web server:** nginx/1.6.2 (fpm-fcgi)

**Database:** mysql 5.5.57

**PHP version:** 5.6.30-0+deb8u1
<details>
        <summary>PHP-modules loaded</summary>

```
 - Core
 - date
 - ereg
 - libxml
 - openssl
 - pcre
 - zlib
 - bcmath
 - bz2
 - calendar
 - ctype
 - dba
 - dom
 - hash
 - fileinfo
 - filter
 - ftp
 - gettext
 - SPL
 - iconv
 - mbstring
 - session
 - posix
 - Reflection
 - standard
 - shmop
 - SimpleXML
 - soap
 - sockets
 - Phar
 - exif
 - sysvmsg
 - sysvsem
 - sysvshm
 - tokenizer
 - wddx
 - xml
 - xmlreader
 - xmlwriter
 - zip
 - cgi-fcgi
 - PDO
 - apcu
 - curl
 - gd
 - json
 - mcrypt
 - memcache
 - memcached
 - mysql
 - mysqli
 - pdo_mysql
 - pdo_sqlite
 - readline
 - sqlite3
 - mhash
 - apc
 - Zend OPcache
```
</details>

### Nextcloud configuration

**Nextcloud version:** 12.0.2 - 12.0.2.0

**Updated from an older Nextcloud/ownCloud or fresh install:** 
YES

**Where did you install Nextcloud from:**
OC 9 =>NC 10,11,12

**Are you using external storage, if yes which one:** Array
(
    [0] => \OC\Files\Storage\Local
)

**Are you using encryption:** no

        <summary>Enabled apps</summary>

```
 - activity: 2.5.2
 - admin_audit: 1.2.0
 - apporder: 0.4.0
 - audioplayer: 2.1.0
 - bookmarks: 0.10.1
 - calendar: 1.5.5
 - checksum: 0.3.5
 - comments: 1.2.0
 - contacts: 1.5.3
 - dav: 1.3.0
 - deck: 0.2.2
 - encryption: 1.6.0
 - federatedfilesharing: 1.2.0
 - federation: 1.2.0
 - files: 1.7.2
 - files_automatedtagging: 1.2.2
 - files_external: 1.3.0
 - files_opds: 0.8.2
 - files_pdfviewer: 1.1.1
 - files_reader: 1.0.4
 - files_sharing: 1.4.0
 - files_texteditor: 2.4.1
 - files_trashbin: 1.2.0
 - files_versions: 1.5.0
 - files_videoplayer: 1.1.0
 - firstrunwizard: 2.1
 - gallery: 17.0.0
 - gpxedit: 0.0.7
 - gpxmotion: 0.0.5
 - gpxpod: 2.2.0
 - issuetemplate: 0.2.2
 - logreader: 2.0.0
 - lookup_server_connector: 1.0.0
 - metadata: 0.5.0
 - news: 11.0.5
 - nextcloud_announcements: 1.1
 - notes: 2.3.1
 - notifications: 2.0.0
 - oauth2: 1.0.5
 - ownpad: 0.5.10
 - password_policy: 1.2.2
 - previewgenerator: 1.0.6
 - provisioning_api: 1.2.0
 - qownnotesapi: 17.5.0
 - rainloop: 5.0.2
 - serverinfo: 1.2.0
 - sharebymail: 1.2.0
 - spreed: 2.0.1
 - survey_client: 1.0.0
 - systemtags: 1.2.0
 - tasks: 0.9.5
 - theming: 1.3.0
 - twofactor_backupcodes: 1.1.1
 - updatenotification: 1.2.0
 - workflowengine: 1.2.0
```
</details>

<details>
        <summary>Disabled apps</summary>

```
 - dashboard
 - spreedme
 - user_external
 - user_ldap
```
</details>

<details>
        <summary>Content of config/config.php</summary>

```
{
    "instanceid": "ocl3mzv8sorh",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "localhost",
        "oc.********.nl"
    ],
    "datadirectory": "\/mnt\/owncloud\/owncloud",
    "overwrite.cli.url": "oc.**********.nl",
    "dbtype": "mysql",
    "version": "12.0.2.0",
    "logtimezone": "UTC",
    "installed": true,
    "memcache.local": "\\OC\\Memcache\\APCu",
    "loglevel": 0,
    "defaultapp": "apporder",
    "knowledgebaseenabled": "true",
    "custom_csp_policy": "default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval'; style-src 'self' 'unsafe-inline' data: blob:; frame-src *; img-src *; font-src 'self' data: blob:; media-src *",
    "maintenance": false,
    "dbname": "ocdatabase",
    "dbhost": "127.0.0.1",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "theme": "",
    "data-fingerprint": "ab51ae68f88b0d5df560447d24ce885f",
    "mail_from_address": "oc",
    "mail_smtpmode": "smtp",
    "mail_domain": "linuxpete.nl",
    "mail_smtpauthtype": "LOGIN",
    "mail_smtphost": "mail.***********.nl",
    "mail_smtpauth": 1,
    "mail_smtpport": "465",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpsecure": "ssl",
    "enabledPreviewProviders": [
        "OC\\Preview\\Image",
        "OC\\Preview\\MP3",
        "OC\\Preview\\TXT",
        "OC\\Preview\\MarkDown",
        "OC\\Preview\\Epub",
        "OC\\Preview\\PDF",
        "OC\\Preview\\OpenDocument",
        "OC\\Preview\\StarOffice",
        "OC\\Preview\\MSOfficeDoc",
        "OC\\Preview\\MSOffice2003",
        "OC\\Preview\\MSOffice2007",
        "OC\\Preview\\FB2"
    ],
    "mysql.utf8mb4": true
}
```
</details>

### Client configuration

**Browser:** IE 11 

**Operating system:** Windows 7, 8

### Logs

<details>
        <summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

<details>
        <summary>Nextcloud log (data/nextcloud.log)</summary>

```
Insert your Nextcloud log here
```
</details>

<details>
        <summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by juliushaertl** [comment]
I have no IE11 instance for testing around, so I will not be able to fix this. PRs are always welcome.

**Comment by linuxpete** [comment]
Sorry, My employer offers me Windows and IE but unfortunately coding isn't my strong point. Is there a way to help each other?

**Comment by juliushaertl** [comment]
@linuxpete One thing you could do is to check the browsers developer tools for any errors in the javascript console. If there is some JS issue that might help to fix it.

**Comment by linuxpete** [comment]
@juliushaertl I will next week on work. At home : linux only ;-)

**Comment by linuxpete** [comment]
@juliushaertl  : Hi hereby the error in English it must be something like : "The Property or Method Includes is not supported by this object". It seems to be a Script438 Eroor

Microsoft writes this about the Error : https://answers.microsoft.com/en-us/windows/forum/windows_8-networking/script438-object-doesnt-support-property-or-method/90df362c-07ef-4b26-9952-f0d9892a1098?auth=1

![apporderdefect](https://user-images.githubusercontent.com/7695965/30799059-e75a986e-a1db-11e7-95fe-53dcbbbfb621.png)

![apporderdefect_1](https://user-images.githubusercontent.com/7695965/30799338-d35ee346-a1dc-11e7-9172-f0cabbfa11b5.png)

**Comment by alx-tuilmenau** [comment]
I changed 2 lines in js/apporder.js, so it will work with IE11, using indexOf instead of includes function:
 
Line 13 to from `if(hidden.includes(id)) {` to `if (hidden.indexOf(id)>-1){`
Line 21 to `return !(hidden.indexOf(e)>-1);`

There is another small problem with IE11, it won't show icons on personal page (with SSL redirection error message), but that's not important enough for me to search the problem

**Comment by juliushaertl** [comment]
@alx-tuilmenau Mind to open a pull request for that change? That would be awesome.

**Comment by juliushaertl** [comment]
Should be fixed with #53
